discordactions/README.md

```diff
@@ -0,0 +1,103 @@
+# Discord actions
+
+
+## **Requests**
+
+|                         Route                          |          Description          |
+|:------------------------------------------------------:|:-----------------------------:|
+|           [GET /groups](#get-groups)           |    Returns all group roles     |
+|          [POST /groups](#post-groups)          |     Creates new group role     |
+| [POST /roles](#post-roles) | Adds a group role to a user |
+
+## **GET /groups**
+
+> returns all group roles
+
+## Response
+
+```json
+{
+    "message":"Roles fetched successfully!",
+    "groups":[
+        {
+            "id":"BcfrtGo9hAhVxcDZ5ADQ",
+            "date":{
+                "_seconds":1683970114,
+                "_nanoseconds":286000000
+            },
+            "createdBy":"k15z2SLFe1U2J3gshXUG",
+            "rolename":"group-peer-programming",
+            "roleid":"1106875628000653332"
+        },
+        {
+            "id":"ELqELhWrdoSX1AusBwPw",
+            "date":{
+                "_seconds":1683917265,
+                "_nanoseconds":874000000
+            },
+            "createdBy":"1uilsoHQZ1TzcYyy25BA",
+            "rolename":"group-live-site",
+            "roleid":"1106653965799657533"
+        },
+    ]
+}
+
```

Comment:
add a fail or 404 response, and a 500 response

Reply:
fixed it